test/common/http/access_log/access_log_formatter_test.cc

```diff
@@ -140,6 +141,23 @@ TEST(AccessLogFormatterTest, requestInfoFormatter) {
     EXPECT_CALL(requestInfo, upstreamHost()).WillOnce(Return(nullptr));
     EXPECT_EQ("-", upstream_format.format(header, header, requestInfo));
   }
+
+  {
+    RequestInfoFormatter upstream_format("UPSTREAM_CLUSTER");
+    std::shared_ptr<Upstream::MockHostDescription> host(new Upstream::MockHostDescription());
+    Upstream::MockClusterInfo cluster;
+    EXPECT_CALL(requestInfo, upstreamHost()).WillRepeatedly(Return(host));
```

Comment:
@mattklein123 @RomanDzhabarov that makes sense. Will add the stuff to MockRequestInfo.